packages/components/src/components/contract-card/contract-card.scss

```diff
@@ -45,8 +45,8 @@
                 grid-template-columns: 1.2fr 1fr;
                 &--turbos {
                     display: flex;
-                    flex-direction: row;
```

Comment:
SCSS property `flex-direction: row; `was deleted, because it's applied by default (initial value).